recycleIT-frontend/src/Components/Services/ServicesList/ServicesSearch/ServicesSearch.tsx

```diff
@@ -39,21 +57,83 @@ const ServicesSearch = (props: IServicesSearchProps) => {
     })
 
     const services = serviceTypes.map((type, i) => {
-        return <MenuItem key={i} value={type.name}>{type.name}</MenuItem>
+        const maxStringLength = 27;
```

Comment:
logic of such formatting can be reused across the project, so it can be moved to utility functions folder for convenience